SPHINXsys/src/shared/particles/base_particles.cpp

```diff
@@ -30,9 +30,13 @@ namespace SPH
 		//----------------------------------------------------------------------
 		//		register particle data
 		//----------------------------------------------------------------------
+		registerAVariable<indexVector, Vecd>(pos_0_, "InitialPosition");
```

Comment:
keep in mind that these indexVectors will be removed in the newest version of the master, again.